### PR TITLE
Add message `url`

### PR DESCRIPTION
**Pull request by remcohaszing**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

It will be rendered by editors.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {a} from './a.js'
 import {an} from './an.js'
 
 const ruleId = 'retext-indefinite-article:retext-indefinite-article'
+const url = 'https://github.com/retextjs/retext-indefinite-article#readme'
```
Comment (ChristianMurphy) [inline lib/index.js:18]: Is this something we could read or construct from the package.json file?

**Review by ChristianMurphy — APPROVED** [review]

File: lib/index.js (hunk 1 above)
Comment (wooorm) [reply lib/index.js:18]: - That would read the whole `package.json` in browsers
- Reading JSON would work in CJS, but currently needs `fs` in Node in ESM

**Review by wooorm — APPROVED** [review]

File: lib/index.js (hunk 1 above)
Comment (ChristianMurphy) [reply lib/index.js:18]: see related discussion in https://github.com/retextjs/retext-contractions/pull/8#discussion_r771552048

**Comment by wooorm** [comment]
Tests are, presumably, failing due to TS? Could you lock it to `~4.4.0`, if that’s iot?

**Comment by wooorm** [comment]
released!

**Pull request merged**